testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -15738,6 +15738,29 @@ private static void checkLogicalOrFunc(SqlOperatorFixture f) {
         true);
   }
 
+  @Test void testBitNotScalarFunc() {
+    final SqlOperatorFixture f = fixture();
+    f.setFor(SqlStdOperatorTable.BITXOR, VmName.EXPAND);
```

Comment:
XOR?
How about a failing test with DECIMAL?

Reply:
Interesting that using XOR had not impact on the tests. Fixed now.

Also added a test for `BITNOT(DECIMAL)`.

Reply:
I think that setFor is just decorative